packages/shared/src/components/plus/PlusComparingCards.tsx

```diff
@@ -55,49 +66,95 @@ const PlusCard = ({
   productOption: plan,
   onClickNext,
 }: PlusCardProps): ReactElement => {
+  const id = useId();
+  const { logSubscriptionEvent } = usePlusSubscription();
+  const { earlyAdopterPlanId, productOptions } = usePaymentContext();
+  const pricingIds = useFeature(feature.pricingIds);
+  const isEarlyAdopterExperiment = !!earlyAdopterPlanId;
+
   const isPaidPlan = !!plan;
+  const cardContentName = isPaidPlan
+    ? OnboardingPlans.Plus
+    : OnboardingPlans.Free;
+  const { heading, features } = cardContent[cardContentName];
+
+  const { hasDiscount, discountPlan } = productOptions.reduce(
```

Comment:
This is quite the thing, it's just needed to showcase the price of the normal one?

Reply:
Yes, we are checking if is free/monthly and then get the discount plan datas 👀